docs/content-services/services/legal-hold.service.md

```diff
@@ -33,19 +35,35 @@ Manages holds for nodes.
     Assign a node to a hold.
     -   _nodeId_: `string` - The Id of the node which will be assigned to a hold
     -   _holdId_: `string` - The Id of the hold to which nodes will be assigned
-    -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`HoldEntry`](../../../lib/js-api/src/api/gs-core-rest-api/docs/HoldEntry.md)`>` - Entry with the hold
+    -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`HoldEntry`](../../../lib/js-api/src/api/gs-core-rest-api/docs/HoldEntry.md)`>` - Entry with the hold <br/>
 
 -   **assignHolds**(nodeIds: `<{id: string}[]>`, holdId: `string`): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`HoldPaging`](../../../lib/js-api/src/api/gs-core-rest-api/docs/HoldPaging.md)`>`<br/>
     Assign a node to a hold.
     -   _nodeIds_: `<{id: string}[]>` - The Ids of the nodes which will be assigned to a hold
     -   _holdId_: `string` - The Id of the hold to which nodes will be assigned
-    -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`HoldPaging`](../../../lib/js-api/src/api/gs-core-rest-api/docs/Hold.md)`>` - Hold paging
+    -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`HoldPaging`](../../../lib/js-api/src/api/gs-core-rest-api/docs/Hold.md)`>` - Hold paging <br/>
 
 -   **unassignHold**(holdId: `string`, nodeId: `string`): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`void`]`>`<br/>
     Assign a node to a hold.
     -   _holdId_: `string` - The hold Id
     -   _nodeId_: `string` - The Id of the node which is unassigned
-    -   **Returns** [`void`]
+    -   **Returns** [`void`] <br/>
+
+-   **bulkHold**(holdId: `string`, query: `string`, language: `string`): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`void`]`>`<br/>
+    Assign a node to a hold.
+    -   _holdId_: `string` - The hold Id
+    -   _query_: `string` - Search query
+    -   _language_: `string` - Language code. `afts` can be used to automatically define a language
+
+    -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`BulkHoldAddResponse`](../../../lib/js-api/src/api/gs-core-rest-api/docs/BulkHoldAddResponse.md)`>` - Bulk status <br/>
+
+-   **bulkFolderHold**(holdId: `string`, query: `string`, language: `string`): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`void`]`>`<br/>
```

Comment:
What is the difference between this method and the one above? From this perspective they look exactly the same

Reply:
add specific query